src/Starcounter.Database.Extensions/PostCommitTransactorContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using Starcounter.Database.ChangeTracking;
+
+namespace Starcounter.Database.Extensions
+{
+    public class PostCommitTransactorContext : ITransactorContext
+    {
+        public List<KeyValuePair<Type, Change>> Changes { get; set; }
```

Comment:
Tuple a better choice over `KeyValuePair`?

Reply:
Also, when this now become public, I'd favor exposing just the read-only `IEnumerable`.